README.md

```diff
@@ -50,37 +52,62 @@ To keep a single config file and aid in running multiple tests at once, RTA read
 utils.setupEnvironmentFromConfigFile('<PATH-TO-CONFIG-FILE>')
 ```
 
- to setup the environment for you. To avoid having to do this in each test file you can setup a global include in your mocha package.json setup like:
+ to setup the environment for you. 
+ 
+ To avoid having to do this in each test file you can setup a global include in your mocha package.json setup like:
 
  ```json
 "mocha": {
-	"timeout": 5000,
-	"file": [
-		"./src/test/include.ts"
-	],
-	"require": [
-		"ts-node/register",
-		"source-map-support/register"
-	]
-}
+    "timeout": 5000,
+    "file": [
+      "./dist/tests/include.js"
+    ],
+    "require": [
+      "source-map-support/register"
+    ]
+  }
 ```
 
-be sure to change the include.ts path to match where you put your include file.
+>> Note: Change the include.ts path to match where you put your include file.
```

Comment:
To me this actually looks harder to see then the previous. Maybe if we did bold instead. The `Note: ` part is a good addition